code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/screwdriver,
+		/obj/item/cable_coil,
+		/obj/item/body_bag)
+
+/obj/item/storage/box/staffkit/eng
+	name = "engineering assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/engineer,
+		/obj/item/clothing/head/helmet/hardhat,
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/mech
+	name = "mechanics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/mechanic,
+		/obj/item/electronics/soldering,
+		/obj/item/device/t_scanner,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/miner
+	name = "mining assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/clothing/head/emerg,
+		/obj/item/clothing/suit/space/emerg,
+		/obj/item/crowbar,
+		/obj/item/mining_tool/power_pick,
+		/obj/item/satchel/mining)
+
+/obj/item/storage/box/staffkit/res
+	name = "research assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffres,
+		/obj/item/device/radio/headset/research,
+		/obj/item/clothing/mask/gas,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/reagent_containers/glass/beaker,
+		/obj/item/device/reagentscanner)
+
+/obj/item/storage/box/staffkit/chef
+	name = "culinary assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffciv,
+		/obj/item/device/radio/headset/civilian,
+		/obj/item/clothing/head/souschefhat,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/satchel/hydro,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/icing,
+		/obj/item/spraybottle/cleaner)
+
+/obj/item/storage/box/staffkit/bar
+	name = "bar assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffciv,
+		/obj/item/device/radio/headset/civilian,
+		/obj/item/clothing/suit/wcoat,
+		/obj/item/clothing/gloves/black,
+		/obj/item/device/reagentscanner,
+		/obj/item/reagent_containers/food/drinks/cocktailshaker,
+		/obj/item/spraybottle/cleaner)
+
+/obj/item/storage/box/staffkit/botany
+	name = "botanical assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffciv,
+		/obj/item/device/radio/headset/civilian,
+		/obj/item/disk/data/cartridge/botanist,
+		/obj/item/clothing/gloves/black,
+		/obj/item/satchel/hydro,
+		/obj/item/device/reagentscanner)
```

Comment:
Add `/obj/item/paper/botany_guide`
Botanists don't even get scanner or satchel on spawn.
Satchels should be available from `GardenGear`

Reply:
Will swap satchel with botany guide, but I think the scanner should stay. Most maps give botanists a few (not directly on them, but close at hand) and having them is pretty important for checking up on plant pot contents.